webapp/src/components/news-item/NewsItem.test.js

```diff
@@ -0,0 +1,139 @@
+import { describe, expect, it } from '@jest/globals';
+import { mount } from '@vue/test-utils';
+import NewsItem from '@/components/news-item/NewsItem';
+
+require('regenerator-runtime/runtime');
+
+
+const TITLE = 'Test-Title';
+const VOTES = 0;
+
+describe('NewsItem', () => {
+
+  it('should set news from parent as props', () => {
+    const title = 'Test-Title';
+    const votes = 0;
+    const wrapper = mount(NewsItem, {
+      propsData: {
+        newsItem: {
+          title: title,
+          votes: votes,
+        },
+      },
+    });
+
+    expect(wrapper.props().newsItem.title).toBe(title);
+    expect(wrapper.props().newsItem.votes).toBe(votes);
+
+  });
```

Comment:
```suggestion
```
This is the already tested behaviour of vue-test-utils. No need to test that again.